tools/CouchDB_MRI_Importer.php

```diff
@@ -1,4 +1,8 @@
 <?php
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once 'generic_includes.php';
+require_once 'CouchDB.class.inc';
+require_once 'Database.class.inc';
```

Comment:
Only the autoload should be required? CouchDB and Database should be autoloaded. (I'm not sure if generic_includes is needed..)